### ISSUE TITLE
No python-vendorize command on Windows

### ISSUE THREAD

**Issue by andy-maier** [mwilliamson/python-vendorize#10]
On Windows, the python-vendorize command is not available after installing the package:
```
> python install vendorize
Collecting vendorize
  Downloading vendorize-0.2.1-py2.py3-none-any.whl (12 kB)
Installing collected packages: vendorize
Successfully installed vendorize-0.2.1

> python-vendorize
'python-vendorize' is not recognized as an internal or external command, operable program or batch file.
```

This could be fixed by supplying a `python-vendorize.bat` batch file like this:
```
@setlocal enableextensions
@set errorlevel=
@python %~d0%~p0%~n0 %*
@endlocal
@exit /b %errorlevel%
```

**Comment by andy-maier** [comment]
PR #11 addresses this as proposed.

**Comment by mwilliamson** [comment]
This should be addressed in the latest release, 0.2.2.
